spec/requests/passwords_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Passwords', type: :request do
+  let(:user) { create :user }
+
+  describe 'POST /create' do
+    it 'resets password' do
+      post '/api/auth/password', params: {
+        email: user.email,
+        redirect_url: 'http://localhost:3000/reset-password'
+      }
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe 'PUT /update' do
+    it 'updates password' do
```

Comment:
Even though this test works, it is good practice to have only 1 `expect` per `it`. Doing so helps a lot when something goes wrong.

If you're confused on how to write good & well formatted tests, please take a look at https://www.betterspecs.org/.